src/providers/twitch/TwitchChannel.hpp

```diff
@@ -226,6 +226,8 @@ class TwitchChannel : public Channel, public ChannelChatters
     void showLoginMessage();
     /** Joins (subscribes to) a Twitch channel for updates on BTTV. */
     void joinBttvChannel() const;
+    void updateSevenTVActivity();
```

Comment:
A comment describing what this is for/what it's used for/when it's used would be nice